client/src/components/CalendarWidget/initial-events.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Are these dummy events meant to stay here? Perhaps they could be changed to something more meaningful for the demonstration such as some public holidays for example

Reply:
In the process of removing these